### ISSUE TITLE
New feature - add HR email address

### ISSUE THREAD

**Issue by muzammildafedar** [muzammildafedar/udayah#3]
### **Description:**

We would like to implement a new feature that allows users to add the HR email address along with the company name and website. This feature will help maintain a list of HR contacts within the application.

### **Feature Requirements:**

> **Popup Form**:
> 
> Create a popup/modal that allows users to enter the following information:
> HR Email Address
> Company Name
> Website

> **Validation**:
> Ensure proper validation for email format and required fields.

> **User Interface**:
> Design a user-friendly interface for the popup/modal.
> Include a "Save" button to store the information.
> State Management:
> 
> Utilize state management (e.g., Provider ) to handle the input data.

Let's discuss the design and flow of this feature further.

**Comment by HMSagar2701** [comment]
Hi  @muzammildafedar, Can I do backend for this ?

**Comment by muzammildafedar** [comment]
> Hi @muzammildafedar, Can I do backend for this ?

Of course, @HMSagar2701, I look forward to your contribution to the backend. Before starting, I would like to request that you set up the MVC structure, Express, ORM, and any other necessary components for the backend, along with proper documentation and architecture. Please create a new issue for the initial backend setup. For backend development, kindly write the code in the functions/ folder. Feel free to share any ideas you have!

**Comment by HMSagar2701** [comment]
Sure, I will start preparing Architecture and I will share with you, what kind of DB we can setup for this, and what all are the endpoints needs to be there, Do we need login page for this Job portal, I want to discuss with you.

**Comment by Sauhard74** [comment]
is the issue open ? if yes I would like to contribute

**Comment by muzammildafedar** [comment]
This has been resolved.
